_content/organization.md

```diff
@@ -189,3 +192,19 @@ Please see the [working groups page]({{ '/content/wg.html' | relative_url }}) fo
 
 
 {{ site.HR }}
+
+<!-- Institutional Board [IB](#institutional-board) and the
```

Comment:
Comment not closed. Again illegal `--` inside comment.

Reply:
Sorry, I'm slowly getting used to this language. Mistakes corrected. Waiting for approving the merge.